claim/migrations/0004_claimattachment.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
             name='ClaimAttachment',
             fields=[
                 ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
-                ('legacy_id', models.IntegerField(blank=True, db_column='LegacyID', null=True)),
+                ('legacy_id', models.UUIDField(blank=True, db_column='LegacyID', null=True)),
```

Comment:
that should not work the primary key for claim is still the id not the uuid 

if you want to do that change you wiull have the update either the claim model to set the uuid as primary key  or the ClaimAttachment model to link the uuid instead of primary key